src/test/resources/application-default.yml

```diff
@@ -0,0 +1,23 @@
+spring:
+  jpa:
+    database: H2
+    hibernate:
+      ddl-auto: create-drop
+    properties:
+      dialect: org.hibernate.dialect.H2Dialect
+      hibernate.format_sql: true
+
+logging:
+  level:
+    org.springframework.orm.jpa: DEBUG
+    org.springframework.transaction: DEBUG
+    org.hibernate.SQL: DEBUG
+    org.hibernate.type.descriptor.sql.BasicBinder: TRACE
+
+dbDriver: jdbc
```

Comment:
not needed ?

Reply:
it's needed: old sheme field held jdbc:h2:mem
now  dbDriver is jdbc and scheme hold h2:mem